### PR TITLE
Fixed 7805 & 7905 to use pin number and IN/OUT instead of VI/VO labels

### PR DESCRIPTION
**Pull request by stanleyseow**

### CODE REVIEW

File: library/regul.lib [hunk 1]
```diff
@@ -45,9 +45,9 @@ F3 "" 0 0 50 H V C CNN
 ALIAS LM7905 LM7912 79L05
 DRAW
 S -200 -150 200 150 0 1 0 N
-X VI VI -400 50 200 R 40 40 1 1 I
-X VO VO 400 50 200 L 40 40 1 1 w
-X GND GND 0 -250 100 U 40 30 1 1 I
+X IN 1 -400 50 200 R 40 40 1 1 I
```
Comment (keruseykaryu) [inline library/regul.lib]: The negative regulators have sligthly different pin numbering. Check the datasheets.

Comment (stanleyseow) [inline library/regul.lib]: Ok, corrected the 7905 pinout

**Comment by diggit** [comment]
Why do you remove modules/packages3d/unused_3d.3dshapes/Device.douille_4mm(black).* files?

Please correct 7805 pinout too.

**78xx**
- 1 VI
- 2 GND
- 3 VO

**79xx**
- 1 GND
- 2 VI
- 3 VO

**Comment by jkriege2** [comment]
This should be fixed by this PR: https://github.com/KiCad/kicad-library/pull/948

**Pull request closed without merging**